lib/laa_crime_forms_common/messages/prior_authority/further_information_request.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "base"
+require "i18n"
+
+module LaaCrimeFormsCommon
+  module Messages
+    module PriorAuthority
+      class FurtherInformationRequest < Base
+        def template
+          "c8abf9ee-5cfe-44ab-9253-72111b7a35ba"
+        end
+
+        def contents
+          {
+            laa_case_reference: case_reference,
+            ufn:,
+            defendant_name:,
+            application_total:,
+            date_to_respond_by:,
+            caseworker_information_requested: comments,
+            date: Time.now.strftime("%-d %B %Y"),
+          }
+        end
+
+      protected
+
+        def comments
+          comments = []
+
+          if further_information_explanation && further_information_explanation.strip != ""
```

Comment:
`further_information_explanation.strip.blank?`?

Reply:
So `blank?` is a Rails thing, and I'm trying to keep this gem as pure Ruby

Reply:
Ah TIL, okay